README.md

```diff
@@ -104,8 +104,7 @@ setFormData({
 });
 ```
 
-Now, we just have one object in state to update whenever an input field
-changes.
+Now, we just have one object in state to update whenever an input field changes.
```

Comment:
In line 115, suggest changing from "...into one more reusable function..." to "...into a single, more reusable, function..."

Also suggest removing "some of" from line 116